3-projects/cyf-ecommerce-api/server.js

```diff
@@ -0,0 +1,267 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
+
+const { Pool } = require("pg");
+
+const pool = new Pool({
+  user: "asif",
+  host: "localhost",
+  database: "cyf_ecommerce",
+  password: "786god",
+  port: 5432,
+});
+
+app.get("/customers", function (req, res) {
+  pool
+    .query("SELECT * FROM customers")
+    .then((result) => {
+      if (result) {
+        res.status(200).json(result.rows);
+      } else res.status(400).send("Couldn't retrieve");
```

Comment:
This would be a server side error so a 5xx error would be better - you can see examples here:https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#5xx_server_errors